backend/models/students.js

```diff
@@ -0,0 +1,22 @@
+import { Model } from 'sequelize';
+
+export default (sequelize, DataTypes) => {
+  class Students extends Model {}
+  Students.init(
+    {
+      name: DataTypes.STRING,
+      email: DataTypes.STRING,
```

Comment:
Can you add the validation for email here?

Also, I think you need to have the `allowNull: false` on `email`, `student_id`, and `name`